### PR TITLE
Handler formatter example was using wrong defined service

### PR DESCRIPTION
**Pull request by gregsomers**
The service class passed to the formatter in the example handler was using the processor service instead of the formatter service. It should instead use `monolog.formatter.session_request`, otherwise monolog complains that the class does not implement interface `Monolog\Formatter\FormatterInterface`.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
This looks OK ... but Monolog was always too confusing to me, so I'll wait for other reviews before merging. Also, if this is the correct change, we should update the XML and PHP config below too. Thanks!

File: logging/processors.rst
Comment (iltar) [inline logging/processors.rst]: This change looks correct to me. Can you also update the other configuration formats? They seem to have the same incorrect value at the formatter option.

**Review by linaori — CHANGES_REQUESTED** [review]

File: logging/processors.rst
Comment (gregsomers) [reply logging/processors.rst]: @iltar no problem, I've updated the other configuration formats

**Review by linaori — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@gregsomers thanks for fixing this! We've merged it on 2.7 branch, which is the oldest maintained branch that contains the bug and we'll "merge it up" on the other branches. GitHub may make it look as closed instead of merged ... but it's merged. Thanks!

**Pull request closed without merging**
